system/fn_enable_schemaview.sql

```diff
@@ -1,26 +1,60 @@
 /*
```

Comment:
I would rename this file system/fn_enable_schemavisible.sql and modify the init script accordingly to avoid naming confusion. Any opinion?